.github/workflows/connector_integration_test_single_dagger.yml

```diff
@@ -27,30 +39,57 @@ jobs:
           aws-secret-access-key: ${{ secrets.SELF_RUNNER_AWS_SECRET_ACCESS_KEY }}
           github-token: ${{ env.PAT }}
           ec2-instance-type: "c5.2xlarge" # https://aws.amazon.com/ec2/instance-types/
-  launch_integration_tests:
+  launch_connector_tests:
     timeout-minutes: 120
-    runs-on: ${{ needs.start-connectors-integration-test-runner.outputs.label }} # run the job on the newly created runner
+    needs: start-runner
+    runs-on: ${{ needs.start-runner.outputs.label }} # run the job on the newly created runner
+    env:
```

Comment:
eventually I want to use SOPS to handle this, but this works for now

Reply:
Could you share what would be done differently with SOPS?

Reply:
We would not load secrets as environment variables here, only GCP service account key. We would store the secrets encrypted as a file in the repository. We would use the GCP service account key to authenticate to GCP to access the key needed to decrypt the secrets, and then we would decrypt the secrets and use them in the workflow as part of the CI run.